scan/debian.go

```diff
@@ -240,18 +240,25 @@ func (o *debian) rebootRequired() (bool, error) {
 
 func (o *debian) scanInstalledPackages() (models.Packages, models.Packages, error) {
 	installed, updatable := models.Packages{}, models.Packages{}
-	r := o.exec("dpkg-query -W", noSudo)
+	r := o.exec("dpkg-query -W -f=\"\\${binary:Package}\\t\\${db:Status-Abbrev}\\t\\${Version}\n\"", noSudo)
 	if !r.isSuccess() {
 		return nil, nil, fmt.Errorf("Failed to SSH: %s", r)
 	}
 
 	//  e.g.
-	//  curl	7.19.7-40.el6_6.4
-	//  openldap	2.4.39-8.el6
+	//  curl	ii 	7.47.0-1ubuntu2.2
+	//  openssh-server	rc 	1:7.2p2-4ubuntu2.2
 	lines := strings.Split(r.Stdout, "\n")
 	for _, line := range lines {
 		if trimmed := strings.TrimSpace(line); len(trimmed) != 0 {
-			name, version, err := o.parseScannedPackagesLine(trimmed)
+			name, status, version, err := o.parseScannedPackagesLine(trimmed)
+			installStatus := status[1]
+			// n -- not-installed
+			// c -- config-files
+			if installStatus == 'n' || installStatus == 'c' {
```

Comment:
+			if installStatus == 'n' || installStatus == 'c' {
```
if installStatus != "I" 
```

I think it better to check with i or not. I think that there are cases where characters other than n, c, i come.